service/storage_proxy.cc

```diff
@@ -1444,7 +1444,7 @@ void storage_proxy_stats::stats::register_stats() {
     _metrics.add_group(COORDINATOR_STATS_CATEGORY, {
         sm::make_histogram("read_latency", sm::description("The general read latency histogram"),
                 {storage_proxy_stats::current_scheduling_group_label()},
-                [this]{ return estimated_read.get_histogram(16, 20);}),
+                [this]{ return estimated_read.get_histogram(std::chrono::microseconds(500), 33, 1.3);}),
```

Comment:
I think we need a few more buckets, so a 4-second query isn't shown as +inf.

Reply:
maybe a better ratio is sqrt(2) or cuberoot(2), so the powers of two are still represented. It's not really important, but it's nicer to see numbers such as 1ms or 16ms. Really a cosmetic issue.

Reply:
I'm going to drop all that ratio as soon as I'll move to the new implementation based on your comments on the previous review.